pkg/apis/core/validation/validation_test.go

```diff
@@ -25716,16 +25716,6 @@ func TestValidatePodResize(t *testing.T) {
 			old:  mkPod(getResources("100m", "100Mi", "", ""), getResources("200m", "200Mi", "", "")),
 			new:  mkPod(core.ResourceList{}, core.ResourceList{}),
 			err:  "Pod QOS Class may not change as a result of resizing",
-		}, {
-			test: "windows pod, no resource change",
-			old:  mkPod(core.ResourceList{}, getResources("100m", "0", "1Gi", ""), podtest.SetOS(core.Windows)),
-			new:  mkPod(core.ResourceList{}, getResources("100m", "0", "1Gi", ""), podtest.SetOS(core.Windows)),
-			err:  "Forbidden: windows pods cannot be resized",
-		}, {
-			test: "windows pod, resource change",
-			old:  mkPod(core.ResourceList{}, getResources("100m", "0", "1Gi", ""), podtest.SetOS(core.Windows)),
-			new:  mkPod(core.ResourceList{}, getResources("200m", "0", "1Gi", ""), podtest.SetOS(core.Windows)),
```

Comment:
I realize this is a revert but maybe we should have a test with `podtest.SetOS(core.Windows)` but with resize working?

Reply:
I can add some